### PR TITLE
fix/add run makemessages for all locales in workflow

### PR DESCRIPTION
**Pull request by pitpalme**
- Add shell script editorconfig
- Modify makemessages to run django4 compatible

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -16,3 +16,7 @@ indent_size = 4
 
 [**.{yml,yaml}]
 indent_size = 2
+
```
Comment (christophmeissner) [inline .editorconfig:19]: what is this file for?

File: makemessages.sh [hunk 2]
```diff
@@ -1,15 +1,44 @@
 #!/usr/bin/env bash
 set -e
+
+default_locale_args() {
+    locale_arg=0
+    while [ $# -gt 0 ]; do
+        arg="${1}"
+        shift
+        if [[ "${arg}" =~ ^--(all|locale)(=|$) ]]; then
```
Comment (christophmeissner) [inline makemessages.sh:9]: Doesn't work with short forms of arguments `-l` and `-a`

File: .github/workflows/pytest.yml [hunk 3]
```diff
@@ -37,7 +37,7 @@ jobs:
       # Check for missing messages in .po files
       - name: Check for missing messages in .po files
         run: |
-          ./makemessages.sh --all
+          ./makemessages.sh
```
Comment (christophmeissner) [inline .github/workflows/pytest.yml:40]: Let's keep `--all` to make it explicit within the workflow, that we want "all" languages, not the current default of the script.

File: .editorconfig (hunk 1 above)
Comment (pitpalme) [reply .editorconfig:19]: Provide basic configuration to editors about how text should be formatted.
Several IDEs evaluate this file to configure workspace accordingly for unified text formatting, e. g. indention, encoding, EOL, etc.
See https://editorconfig.org/

File: .github/workflows/pytest.yml (hunk 3 above)
Comment (pitpalme) [reply .github/workflows/pytest.yml:40]: *meeehhhh* OK. ;)

**Review by christophmeissner — APPROVED** [review]

**Pull request merged**